toolkit/tools/downloader/downloader.go

```diff
@@ -34,14 +34,17 @@ var (
 	tlsClientCert = app.Flag("certificate", "TLS client certificate to use when downloading files.").String()
 	tlsClientKey  = app.Flag("private-key", "TLS client key to use when downloading files.").String()
 
-	dstFile   = app.Flag("output-file", "Destination file to download to").Short('O').String()
-	prefixDir = app.Flag("directory-prefix", "Directory to download to").Short('P').String()
-	srcUrl    = app.Arg("url", "URL to download").Required().String()
+	dstFile     = app.Flag("output-file", "Destination file to download to").Short('O').String()
+	prefixDir   = app.Flag("directory-prefix", "Directory to download to").Short('P').String()
+	rpmFilename = app.Arg("rpm-filename", "RPM to download").Required().String()
+	srcUrls     = app.Arg("url-list", "URLs to download").Required().Strings()
```

Comment:
To avoid confusion, I think we probably want this to be a flag rather than an arg (ie require `--url-list` to be passed explicilty)

Reply:
Agree that it's an improvement.